tests/test_crud_immunisation_api.py

```diff
@@ -170,3 +173,43 @@ def test_bad_nhs_number_nhs_login(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
 
     assert result.status_code == 400
     assert res_body["resourceType"] == "OperationOutcome"
+
+
+@pytest.mark.nhsd_apim_authorization(
+    {
+        "access": "healthcare_worker",
+        "level": "aal3",
+        "login_form": {"username": "656005750104"},
+    }
+)
+def test_get_s_flag_patient(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
+    # Arrange
+    token = nhsd_apim_auth_headers["Authorization"]
+    imms_api = ImmunisationApi(nhsd_apim_proxy_url, token)
+
+    # Act
+    imms = create_an_imms_obj(nhs_number="9449310610")
```

Comment:
There is a set of `valid_nhs_number`s in the the config file. Can you please add it there with something like `s_flag_patient` . Right now it looks like a magic number.